lesson05/script.js

```diff
@@ -0,0 +1,25 @@
+const ul = document.getElementById('js-list');
+const lists = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+   ]
+
+const promise = new Promise(resolve => {
+    resolve(lists)
+});
+
+promise.then(() => {
+    const fragment = document.createDocumentFragment();
+    for(const list of lists){
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+        a.textContent = list.text;
+        a.href = '/' +list.to;
```

Comment:
```suggestion
        a.href = /${list.to}`;
```
nits
You can also write it this way.
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Template_literals

Reply:
Thank you for your comment!
I study template literals. 
I will write using template literals.